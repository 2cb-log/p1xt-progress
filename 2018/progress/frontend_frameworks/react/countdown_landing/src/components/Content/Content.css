.Content {
    text-align: left;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    margin: 100px;
}

.Content__banner {
    font-family: 'Dosis', sans-serif;
    font-weight: bold;
    width: 50%;
}

.Content__details {
    font-family: 'Dosis', sans-serif;
    width: 50%;
}

.Content__notes {
    font-family: 'Roboto', sans-serif;
}