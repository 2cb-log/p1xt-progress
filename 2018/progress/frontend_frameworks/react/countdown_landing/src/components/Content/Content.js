import React from 'react';
import './Content.css';

function Content(props) {
  return (
    <div className="Content">
      Photo by Benjamin Voros on Unsplash
    </div>
  );

}

export default Content;
