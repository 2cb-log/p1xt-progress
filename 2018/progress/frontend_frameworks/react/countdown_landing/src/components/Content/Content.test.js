import React from 'react';
import ReactDOM from 'react-dom';
import Content from './Content';
import renderer from 'react-test-renderer';
describe('<Content />', () => {
  const wrapper = shallow(<Content />);
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Content />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders correctly', () => {
    const tree = renderer
      .create(<Content />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
