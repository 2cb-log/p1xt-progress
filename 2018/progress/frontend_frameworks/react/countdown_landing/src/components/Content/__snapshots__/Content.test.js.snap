// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> renders correctly 1`] = `
<div
  className="Content"
>
  Photo by Benjamin Voros on Unsplash
</div>
`;
